### PR TITLE
Add new hypothesis attaching lines to spliting lines

### PR DESCRIPTION
**Pull request by bendaamerahm**

### CODE REVIEW

File: src/components/graph/menus/network-modification-node-editor.js [hunk 1]
```diff
@@ -588,7 +600,7 @@ const NetworkModificationNodeEditor = () => {
                                 className={classes.list}
                                 onChecked={setSelectedItems}
                                 values={modifications}
-                                itemRenderer={(props) => (
+                                itemRenderer={(props) => {
```
Comment (laurgarn) [inline src/components/graph/menus/network-modification-node-editor.js:603]: Why this change `()` -> `{;}` ?

**Review by laurgarn — APPROVED** [review]

File: src/components/graph/menus/network-modification-node-editor.js (hunk 1 above)
Comment (bendaamerahm) [reply src/components/graph/menus/network-modification-node-editor.js]: my bad i was using a console.log to test an forgot to clean the code.

File: src/components/dialogs/lines-attach-to-split-lines-dialog.js [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+ * @param substationOptionsPromise Promise handling list of network substations
+ * @param editData the possible line split with voltage level creation record to edit
+ */
+const LinesAttachToSplitLinesDialog = ({
+    open,
+    onClose,
+    lineOptionsPromise,
+    voltageLevelOptionsPromise,
+    currentNodeUuid,
+    substationOptionsPromise,
```
Comment (FranckLecuyer) [inline src/components/dialogs/lines-attach-to-split-lines-dialog.js:50]: substationOptionsPromise is not used

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/components/dialogs/lines-attach-to-split-lines-dialog.js (hunk 2 above)
Comment (bendaamerahm) [reply src/components/dialogs/lines-attach-to-split-lines-dialog.js]: fixed

**Review by FranckLecuyer — APPROVED** [review]
